src/couch/src/couch_att.erl

```diff
@@ -680,11 +680,20 @@ is_compressible(Type) ->
 
 
 max_attachment_size() ->
-    case config:get("couchdb", "max_attachment_size", "infinity") of
-        "infinity" ->
-            infinity;
-        MaxAttSize ->
-            list_to_integer(MaxAttSize)
+    ConfigMaxAttSize = config:get_integer(
+                            "couchdb",
+                            "max_attachment_size",
+                            ?ATT_SIZE_LIMIT_BYTES
+                        ),
```

Comment:
Was this indentation intentional? I only ask because it's neither the current style of generally indenting 4 spaces from the previous line, nor I think really `erlfmt` style.